### ISSUE TITLE
GUI: Close ActiveMQ threads on switch instrument

### ISSUE THREAD

**Issue by John-Holt-Tessella** [ISISComputingGroup/IBEX#1806]
The threads for active MQ do not seem to be closed after switching instrument. To reproduce:

1. Open GUI
1. Switch Instrument 3 or 4 times
1. Look at debug view in eclipse (multiple threads connected to the different instruments)

**Comment by DominicOram** [comment]
Note that work on https://github.com/ISISComputingGroup/IBEX/issues/1756 will also be using ActiveMQ.

**Comment by John-Holt-Tessella** [comment]
Make sure you update IOCLogServer first (because of connections to active MQ from another ticket that was missed)

**Comment by Tom-Willemsen** [comment]
Do after #1756
